content/pt/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: blog
+title: "Não entre em pânico: Kubernetes e Docker"
+date: 2020-12-02
+slug: dont-panic-kubernetes-and-docker
+---
+
+**Autores / Autoras**: Jorge Castro, Duffie Cooley, Kat Cosgrove, Justin Garrison, Noah Kantrowitz, Bob Killen, Rey Lejano, Dan “POP” Papandrea, Jeffrey Sica, Davanum “Dims” Srinivas
+
+**Tradução:** João Brito (Getup)
```

Comment:
why do we need to add the company as part of the name? the authors names do not have the company attached to that.

Reply:
I've follow the korean translate example, but it's not important to me, i'm just trying :D

Reply:
+1, I guess that the company name can be removed :)

Reply:
It's also OK to mention a company name, even if the original authors did not.

Reply:
Removed